CRM/Contribute/BAO/Contribution.php

```diff
@@ -2639,7 +2639,7 @@ public function _gatherMessageValues($input, &$values, $ids = array()) {
       }
       // set lineItem for contribution
       if ($this->id) {
-        $lineItem = CRM_Price_BAO_LineItem::getLineItems($this->id, 'contribution', 1);
+        $lineItem = CRM_Price_BAO_LineItem::getLineItemsByContributionID($this->id, TRUE);
```

Comment:
Looks like my attempt above broke the tests. I have a suspicion it is either a leakage problem or the tests might need fixing. Regardless the way to not get blocked is to hold off on changing this line & comment that it should be updated to not require the isQuick - that way at least this PR can be merged